src/main/java/com/jaspersoft/jasperserver/jaxrs/client/dto/reports/ReportExecutionRequest.java

```diff
@@ -119,8 +113,8 @@ public String getOutputFormat() {
         return outputFormat;
     }
 
-    public ReportExecutionRequest setOutputFormat(String outputFormat) {
-        this.outputFormat = outputFormat;
+    public ReportExecutionRequest setOutputFormat(ReportOutputFormat outputFormat) {
```

Comment:
Please keep a method with String argument. Let them be overloaded. JRS can support custom export formats and we have to provide an ability to get export in this format without rebuilding of a client.